api/service.go

```diff
@@ -347,7 +349,12 @@ func (s *service) AddProject(ctx context.Context, req *pb.AddProjectRequest) (*p
 		log.Fatal("scope required")
 	}
 
-	proj, err := s.collections.Projects.Create(ctx, req.Name, scope)
+	addr, err := s.fcClient.Wallet.NewWallet(ctx, "bls")
```

Comment:
Ok, so not sure how tightly coupled we want project creation to be to filecoin. Right now, it's very tightly coupled: The `fcClient` must exist and it must be pointed at a running filecoin grpc service. Do we want to make this more flexible?

Reply:
Maybe for now we could just check to see if the `fcClient` is `nil`, if not, then create the address. That would be nice for development / tests. In the future, I could see it being a requirement. Dunno though.

Reply:
sounds good. required a little special handling in `GetProject` as well to only make the query for balance if the fc account != "", but done